installer/lib/phx_new/web.ex

```diff
@@ -48,6 +48,20 @@ defmodule Phx.New.Web do
     {:eex,  "phx_test/views/page_view_test.exs",              :web, "test/:web_app/views/page_view_test.exs"},
   ]
 
+  template :live, [
+    {:eex, "phx_live/templates/layout/root.html.leex", :web, "lib/:web_app/templates/layout/root.html.leex"},
+    {:eex, "phx_live/templates/layout/app.html.leex",  :web, "lib/:web_app/templates/layout/app.html.leex"},
+    {:eex, "phx_live/templates/layout/live.html.leex", :web, "lib/:web_app/templates/layout/live.html.leex"},
+    {:eex, "phx_live/templates/page/home.html.leex",   :web, "lib/:web_app/templates/page/home.html.leex"},
```

Comment:
I mentioned this in the template itself, but we can probably re-use the same "page/index.html.eex" as long as we wrap the live view specific code in a conditional. This way it will be less things to us maintain. :)

Reply:
Agree. I wasn't sure what direction we were going to go content wise, but if it's 95% the same, then it makes sense to share